wagtailcodeblock/templatetags/wagtailcodeblock_tags.py

```diff
@@ -79,13 +79,6 @@ def load_prism_css():
             """prism-line-numbers.min.css" rel="stylesheet">"""
         )
 
-    if get_copy_to_clipboard():
-        toolbar = True
-        script += (
-            f"""<link href="{PRISM_PREFIX}{PRISM_VERSION}/plugins/copy-to-clipboard/"""
-            """prism-copy-to-clipboard.min.css" rel="stylesheet">"""
-        )
-
```

Comment:
Since we're removing this, we may be able to remove the function from the import, and the settings file. I'll check in depth on Monday. Thanks so much for the PR!

Reply:
We only removed the CSS for it since there isn't one for it on CDNJS and was causing the 404. But the JS functionality still works and probably for that reason we would want to keep the function. My pleasure, and thank you for putting this out there!